core/service/transactionQueue.ts

```diff
@@ -75,9 +75,9 @@ class TransactionQueue<T extends IAsset> implements ITransactionQueueService<T>
 
     // TODO can be optimized if check senderId and recipientId
     reshuffle(): void {
-        while (this.getSize().conflictedQueue > 0) {
-            this.push(this.conflictedQueue.pop().transaction);
-        }
+        this.queue.push(...this.conflictedQueue.map(obj => obj.transaction));
```

Comment:
Return previous logic. New logic boken. You replaced this.push on this.queue.push. It`s not same logic.